gcm_drivers.slurm

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+#SBATCH --job-name=gcm_munge   	# name that you chose
+#SBATCH --partition=cpu         	# the partition you want to use, for this case prod is best
+#SBATCH --cpus-per-task=2		# Number of cores (None of this is actually parallelized)
+#SBATCH --nodes=1			# Number of nodes
+#SBATCH -A iidd                 	# List account
+#SBATCH --output=gcm_munge.out	# file to save output messages to
+#SBATCH --time=20:00:00        		# time at which the process will be cancelled if unfinished
+#SBATCH --mem=192GB
+#SBATCH --mail-type=ALL
+#SBATCH --mail-user=lplatt@usgs.gov
+
+module load singularity
+
+ulimit -u 1541404
+
+srun singularity exec \
+    lake-temperature-model-prep.sif \
+    Rscript gcm_download.R
+
```

Comment:
In order to lose the `gcm_download.R`, you should be able to replace this with

```bash
srun singularity exec Rscript -e "library(scipiper); scmake('7_drivers_munge/out/7_GCM_driver_files.ind')"
```

Reply:
So I don't need the `lake-temperature-model-prep.sif` line?

Reply:
You most definitely do need it, sorry about that!
```bash
srun singularity exec \
    lake-temperature-model-prep.sif \
    Rscript -e "library(scipiper); scmake('7_drivers_munge/out/7_GCM_driver_files.ind')"
```

Reply:
OK, I did keep it in my two additional commits. Ready for review of those!